src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -886,6 +913,9 @@ export class EditDataGridPanel extends GridParentComponent {
 				scrollTop: viewport.scrollTop,
 				scrollLeft: viewport.scrollLeft
 			};
+			this.telemetryService.createActionEvent(TelemetryKeys.TelemetryView.EditDataGrid, TelemetryKeys.TelemetryAction.EditSaveViewState)
+				.withAdditionalProperties({ gridSelections: this.savedViewState.gridSelections, scrollLeft: this.savedViewState.scrollLeft, scrollTop: this.savedViewState.scrollTop })
```

Comment:
This likely isn't going to be what you want - we don't stringify arrays by default so it'll just end up as some useless type name